core/prob_dist.py

```diff
@@ -187,13 +187,11 @@ def freq_list(self):
         return [self.freq_dict[s] for s in self.alphabet]
 
     @property
-    @cache
```

Comment:
Can you remind me the issue with adding this cache and why we decided to remove it?